rskj-core/src/test/java/co/rsk/net/messages/CodeRequestMessageTest.java

```diff
@@ -0,0 +1,41 @@
+package co.rsk.net.messages;
+
+import co.rsk.core.RskAddress;
+import co.rsk.crypto.Keccak256;
+import org.ethereum.TestUtils;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+public class CodeRequestMessageTest {
+    private static Keccak256 blockHash;
+    private static RskAddress address;
+    private static CodeRequestMessage message;
+    private byte[] hashBytes = TestUtils.randomBytes(20);
+
+    @Before
+    public void setUp() {
+        blockHash = mock(Keccak256.class);
+        address = mock(RskAddress.class);
```

Comment:
You're mocking an address and then mocking a method you only use yourself, replace the address.getBytes() line with hashBytes.

Also, please create a new randomBytes hash for the address, so it's different than the blockHash